### ISSUE TITLE
STORIES

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#3173]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.67.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.66.4...4.67.0) (2024-01-10)

- Release 4.67.0 [`5fc7502`](https://github.com/open-wa/wa-automate-nodejs/commit/5fc7502d96eddd4898e1224a727ed5f306d0d31f)
- 🚑 remove cmd listener code #3161 [`a63a969`](https://github.com/open-wa/wa-automate-nodejs/commit/a63a969b6bedd3218669da6091fc19306c7cd305)
- updated types-only package [`6636245`](https://github.com/open-wa/wa-automate-nodejs/commit/6636245a31bc5c09d09bf556765a880b6e3b4ce2)
- Create CNAME [`4916cac`](https://github.com/open-wa/wa-automate-nodejs/commit/4916cac8589f66ff9948e8142e14685c693e1ba0)

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.67.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.66.4...4.67.0) (2024-01-10)

- Release 4.67.0 [`5fc7502`](https://github.com/open-wa/wa-automate-nodejs/commit/5fc7502d96eddd4898e1224a727ed5f306d0d31f)
- 🚑 remove cmd listener code #3161 [`a63a969`](https://github.com/open-wa/wa-automate-nodejs/commit/a63a969b6bedd3218669da6091fc19306c7cd305)
- updated types-only package [`6636245`](https://github.com/open-wa/wa-automate-nodejs/commit/6636245a31bc5c09d09bf556765a880b6e3b4ce2)
- Create CNAME [`4916cac`](https://github.com/open-wa/wa-automate-nodejs/commit/4916cac8589f66ff9948e8142e14685c693e1ba0)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`npm set "//registry.npmjs.org/:_authToken=$NPM_TOKEN"`);
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
  })();
```
</details>

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.68.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.67.0...4.68.0) (2024-02-19)

- 🔧 fix package lock [`2a36030`](https://github.com/open-wa/wa-automate-nodejs/commit/2a36030c8ef5d3e4bd0837aa6214b53dde76c1c3)
- ⚡️ remote access now uses cloudflare tunnels #3170 [`4df62c4`](https://github.com/open-wa/wa-automate-nodejs/commit/4df62c444bfc2177fde6b4c1322170fe8f1f980c)
- Release 4.68.0 [`68bd4b5`](https://github.com/open-wa/wa-automate-nodejs/commit/68bd4b50c9fe3028d4c4a83846aeb7376a21a4fd)
- ✨ feat: sending text, image and video stories now enabled for restricted license holders #3173 [`705a232`](https://github.com/open-wa/wa-automate-nodejs/commit/705a2328ab03f430b90751d38fdd8f2e6964738d)
- 🚑 attempt fix for #3167 [`2531781`](https://github.com/open-wa/wa-automate-nodejs/commit/2531781626afcb22e754e8679cd57cfe0bb3f4ff)
- 🎨 chatwoot integration improvements #3171 [`7c56729`](https://github.com/open-wa/wa-automate-nodejs/commit/7c56729c5a9f1371b35387d0ef4998d98aeb5dd6)
- Update comment-run.yml [`89d686d`](https://github.com/open-wa/wa-automate-nodejs/commit/89d686dbd580194508050a225e6626917ed24450)
- Update comment-run.yml [`1be18fc`](https://github.com/open-wa/wa-automate-nodejs/commit/1be18fcfc6370d4717776ca09657bf38c9634c3f)
- Update comment-run.yml [`07c0c3c`](https://github.com/open-wa/wa-automate-nodejs/commit/07c0c3c5261ec7e05b21db4fe06208082b31a1e1)
- Update comment-run.yml [`a1f931e`](https://github.com/open-wa/wa-automate-nodejs/commit/a1f931ebd9650509119f203fb5a8884b53bb3f0a)
- 🇵🇸: ADDED https://techforpalestine.org/ BADGE [`80c5592`](https://github.com/open-wa/wa-automate-nodejs/commit/80c5592b573ccfb048763fe83afdd29136338d80)
- updated types-only package [`02c6cec`](https://github.com/open-wa/wa-automate-nodejs/commit/02c6cec5f861381e065fab56079471046a364f34)
